tests/integration/serverless.yml

```diff
@@ -12,6 +12,8 @@ provider:
     DD_API_KEY: ${env:DD_API_KEY}
   lambdaHashingVersion: 20201221
   timeout: 15
+  deploymentBucket:
+    name: integration-tests-deployment-bucket
```

Comment:
Can you link to docs for `deploymentBucket`? I don't see where it sets up the 1 day lifecycle policy by default.

Reply:
The lifecycle policy is a setting in AWS. I hardcoded this bucket name in the test, and then configured the bucket in AWS to delete all objects older than one day.

Does that answer your question or do you still want to see docs for something?

Reply:
Nah, we're good. I misunderstood you, I thought `deploymentBucket` was setting the lifecycle policy automatically. It looks like it automatically prunes artifacts once it has more than 5 artifacts, which is pretty good in and of itself. https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/